src/wallet/bdk_wallet/wallet.rs

```diff
@@ -25,3 +36,191 @@ pub struct Wallet {
     rpc: Client,
     wallet_file_path: PathBuf,
```

Comment:
We should also add the `bdk_store` in this PR itself. That will change the init function and we don't wanna tackle that later in a separate PR. Let's finish all we need to do in the init function here itself.